blocks/default-output-block/output-types/default.jsx

```diff
@@ -106,6 +106,7 @@ const SampleOutputType = ({
 		querylyId,
 		querylyOrg,
 		locale,
+		textDirection,
```

Comment:
We should have this default to `ltr`, so we don't end up with an empty `dir` attribute.

Reply:
good catch, I went ahead and added the default as well as a test to check for the default when no textDirection is passed in